hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/container/replication/ReplicationTestUtil.java

```diff
@@ -84,14 +84,27 @@ public static Set<ContainerReplica> createReplicas(ContainerID containerID,
     return replicas;
   }
 
+  public static Set<ContainerReplica> createReplicas(ContainerID containerID,
+      ContainerReplicaProto.State replicaState, long keyCount, long bytesUsed,
+      int... indexes) {
+    Set<ContainerReplica> replicas = new HashSet<>();
+    for (int i : indexes) {
+      DatanodeDetails dn = MockDatanodeDetails.randomDatanodeDetails();
+      replicas.add(createContainerReplica(containerID, i, IN_SERVICE,
+          replicaState, keyCount, bytesUsed,
+          dn, dn.getUuid()));
+    }
+    return replicas;
+  }
+
   public static Set<ContainerReplica> createReplicasWithSameOrigin(
       ContainerID containerID, ContainerReplicaProto.State replicaState,
       int... indexes) {
     Set<ContainerReplica> replicas = new HashSet<>();
     UUID originNodeId = MockDatanodeDetails.randomDatanodeDetails().getUuid();
     for (int i : indexes) {
       replicas.add(createContainerReplica(
-          containerID, i, IN_SERVICE, replicaState,
+          containerID, i, IN_SERVICE, replicaState, 123L, 1234L,
```

Comment:
It's better to avoid magic numbers even in test code. I suggest adding local constants for these numbers.